packages/components/psammead-brand/src/index.jsx

```diff
@@ -54,7 +57,12 @@ const Banner = styled.div`
   }
 
   @media (max-width: ${GEL_GROUP_0_SCREEN_WIDTH_MAX}) {
-    min-height: ${({ svgHeight }) => svgHeight / 16 + 5}rem;
+    ${({ scriptLink, svgHeight }) =>
+      scriptLink !== null &&
```

Comment:
Thanks for the changes. One last thing:

```suggestion
      scriptLink &&
```